status-server/src/public/js/client.js

```diff
@@ -42,15 +42,77 @@ function init() {
     }
   }
   // populate the LEDs with some initial colors
-  var colors = ['#ffff00', '#ffff00', '#00ffff', '#00ff07', '#772cb6'];
+  var colors = ['#ff00ff', '#ffff00', '#00ffff', '#00ff07', '#772cb6'];
   colors.forEach((color, idx) => {
     var node = document.getElementById("led" + idx);
     var ctx = node.getContext("2d");
     ctx.fillStyle = color;
     ctx.fillRect(0,0,300,300);
+    ctx.font = "10px Arial";
+    ctx.strokeText(idx ,node.width/2,node.height/2);
+
+
   });
 }
+function animate(ct){
+  var fetchResponse = window.fetch('http://localhost:3000/status'+ct+'.json'+Date.now()).then(
```

Comment:
for the fetch function you have just to put the URL the type of request Post or Get it's optional to define as I know from here https://davidwalsh.name/fetch yeah we could know about the failure and success with some console.log (y)